sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -1620,6 +1620,8 @@ def get_transceiver_status(self):
         rxoutput_status_hostlane6    = BOOLEAN                          ; rx output status on host lane 6
         rxoutput_status_hostlane7    = BOOLEAN                          ; rx output status on host lane 7
         rxoutput_status_hostlane8    = BOOLEAN                          ; rx output status on host lane 8
+        tx_disable                   = BOOLEAN                          ; tx disable status
```

Comment:
Can we remove both tx_disable/tx_disabled_channel assignments and comments from get_transceiver_bulk_status for all types of SFPs?
Also, can you please ensure we set both tx_disable and tx_disabled_channel for non-CMIS transceivers too as part of get_transceiver_status?

Reply:
1. yes, both tx_disable/tx_disabled_channel assignments and comments from get_transceiver_bulk_status for all types of SFPs. (assignments were already removed, and baseline code has no table description comment in get_transceiver_bulk_status implementation)
2. get_transceiver_status API is only applicable/available for CMIS/C-CMIS.

Reply:
@longhuan-cisco - For #1, I still see tx_disable and tx_disabled_channel being assigned in get_transceiver_bulk_status 
 - cmis.py. Can you please help in looking into it further?
 
 For #2, I thought we were planning to display the below fields for all types of xcvrs through get_transceiver_status  API. @prgeor @longhuan-cisco @qinchuanares - Please correct me if I am wrong.
 1.	Tx Disable
2.	Tx Disable Channel
3.	Rx LOS
4.	Tx Fault

Reply:
Agreed. Added status API for non-cis